tasks/collectors/collect-collectors-resources/tests/test-collect-collectors-data-fail-empty-previous-release-value.yaml

```diff
@@ -0,0 +1,112 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-collect-collectors-data-fail-empty-previous-release-value
+spec:
+  description: |
+    Run the collect-collectors-resources task without a value for the PreviousRelease cr and verify that
+    the task fails as expected.
```

Comment:
It doesn't fail, does it? It just saves an empty json.

Reply:
yes. renamed test and updated description.